code/modules/clothing/head/helmet.dm

```diff
@@ -74,6 +74,15 @@
 /obj/item/clothing/head/helmet/sec
 	can_flashlight = TRUE
 
+/obj/item/clothing/head/helmet/syndicate
+	can_flashlight = TRUE
+	name = "syndicate helmet"
+	desc = "Standar issue security helmet. Painted red to help with identification,additionally fitted with flash protection and muffling"
+	icon_state = "syndicate_helmet"
+	item_state = "syndicate_helmet"
+	can_flashlight = TRUE
```

Comment:
double `can_flashlight`

Reply:
Adressed the issue thank you for pointing this out